exercises/concept/cache/cache_struct.go

```diff
@@ -0,0 +1,31 @@
+package cache
```

Comment:
Instead of "cache_struct.go", "types.go" is a more common name for a file like this.